src/main/java/edu/bu/metcs673/trackr/domain/BankAccount.java

```diff
@@ -0,0 +1,42 @@
+package edu.bu.metcs673.trackr.domain;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "BANK_ACCOUNT")
+public class BankAccount {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private long id;
+
+	@ManyToOne
+	@JoinColumn(name = "userId", referencedColumnName = "id", nullable = false)
+	private User user;
+
+	@Column(nullable = false, length = 20)
+	private String accountType;
+
+	@Column
+	private String accountDescription;
+
+	@Column(nullable = false, precision = 2)
+	private double balance;
```

Comment:
I would use BigDecimal for handling money to ensure we don't lose precision. I just looked up this.
https://stackoverflow.com/questions/8148684/what-data-type-to-use-for-money-in-java